addons/mrp/models/mrp_production.py

```diff
@@ -514,6 +515,16 @@ def _compute_show_lot_ids(self):
         ('qty_positive', 'check (product_qty > 0)', 'The quantity to produce must be positive!'),
     ]
 
+    @api.depends('qty_producing', 'date_planned_start')
+    def _compute_forecasted_issue(self):
```

Comment:
It is correct in this case:
- You have a virtual_available < 0 for days+2. virtual_available = -2
- Then you create a MO (draft) with `product_qty` = 2 to fulfill at days+2
- If the MO is in a draft state (no move_finished_ids yet), then the virtual_available is still -2
- Then the `forecasted_issue` is True and the icon is red. True ?

Then my question is "Shouldn't be green in that case" (MO still draft) ?

Reply:
Good point, I agree it should be green in that case, I'll change it (same for PO)